stresstests/src/test/java/org/neo4j/causalclustering/stresstests/BackupLoad.java

```diff
@@ -30,43 +28,37 @@
 import org.neo4j.helper.IsConnectionException;
 import org.neo4j.helper.IsConnectionRestByPeer;
 import org.neo4j.helper.IsStoreClosed;
-import org.neo4j.causalclustering.discovery.Cluster;
 
-class BackupLoad extends RepeatUntilOnSelectedMemberCallable
+class BackupLoad extends RepeatOnRandomMember
 {
-    private final Predicate<Throwable> isTransientError =
-            new IsConnectionException().or( new IsConnectionRestByPeer() ).or( new IsChannelClosedException() )
-                    .or( new IsStoreClosed() );
+    private final Predicate<Throwable> isTransientError = new IsConnectionException()
+            .or( new IsConnectionRestByPeer() )
```

Comment:
I know this wasn't caused by this PR, but could maybe roll in a refactor for this typo?